CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ option(SSG_PRODUCT_WRLINUX1019 "If enabled, the WRLinux1019 SCAP content will be
 option(SSG_CENTOS_DERIVATIVES_ENABLED "If enabled, CentOS derivative content will be built from the RHEL content" TRUE)
 option(SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED "If enabled, Scientific Linux derivative content will be built from the RHEL content" TRUE)
 
+option(VALIDATE_EVERYTHING "If enabled, include all validation tests regardless of oscap version requirements" FALSE)
```

Comment:
I got confused by the help text and thought that no content was validated by default until I read code of `function(define_validate_product PRODUCT)` or commit message.

I suggest to updated the help text to clarify that validation of certain content is dependent on the scanner version, and include OVERRIDE or FORCE in the variable name.